docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana either through the UI or the Grafana config file.
+
+Grafana SAML UI has the following advantages over configuring SAML in the Grafana config file:
+
+- It is accessible by hosted Grafana users;
+- It doesn't require Grafana to be restarted after a configuration update;
+- Access to SAML UI only requires access to SAML settings, so it can be used by users with limited access to Grafana's configuration.
```

Comment:
```suggestion
- Access to the SAML UI only requires access to SAML settings, so it can be used by users with limited access to Grafana's configuration.
```